cookbook/doctrine/file_uploads.rst

```diff
@@ -4,6 +4,13 @@
 How to Handle File Uploads with Doctrine
 ========================================
 
+.. note::
+
+    Instead of handling file uploading yourself, you may consider using the
+    `VichUploaderBundle`_ community bundle. This bundle provides all the common
+    operations (such as file renaming, saving and deleting) and it's tightly
+    integratd with Doctrine ORm, ODM, PHPCR ODM and Propel.
```

Comment:
typo: ``integrated``.

And ``ORM`` should be all uppercase.
And ``ODM`` should be replaced by ``MongoDB ODM`` (there are several Doctrine ODMs)